src/app/_models/targets.ts

```diff
@@ -0,0 +1,37 @@
+import * as am4charts from '@amcharts/amcharts4/charts';
+
+export enum TargetFieldName {
+  THREE_D = 'three_d',
+  HQ = 'hq',
```

Comment:
Just as mentioned in the backend review, it is better to change this to `high_quality` instead

Reply:
Done